partially reverts a commit called "Fix many review comments." that messes with
how cargo auditable doesn't properly handle this yet

--- a/crates/arti-relay/Cargo.toml
+++ b/crates/arti-relay/Cargo.toml
@@ -16,6 +16,8 @@ all-features = true
 
 [features]
 full = [
+    "metrics",
+    "opentelemetry",
     "fs-mistrust/full",
     "safelog/full",
     "tor-async-utils/full",
@@ -57,13 +59,13 @@ experimental = ["metrics", "opentelemetr
 
 metrics = ["tor-chanmgr/metrics", "dep:metrics-exporter-prometheus", "__is_experimental"]
 opentelemetry = [
-    "dep:tracing-opentelemetry",
+    "tracing-opentelemetry",
     "dep:opentelemetry",
-    "dep:otlp-file-exporter",
-    "dep:opentelemetry_sdk",
-    "dep:opentelemetry-otlp",
-    "dep:opentelemetry-appender-tracing",
-    "dep:opentelemetry-proto",
+    "otlp-file-exporter",
+    "opentelemetry_sdk",
+    "opentelemetry-otlp",
+    "opentelemetry-appender-tracing",
+    "opentelemetry-proto",
     "__is_experimental",
 ]
 __is_experimental = []
